SPECS/pcre2/pcre2.spec

```diff
@@ -1,13 +1,13 @@
 Summary:        A library for Perl-compatible regular expressions
 Name:           pcre2
-Version:        10.34
-Release:        2%{?dist}
+Version:        10.39
+Release:        1%{?dist}
 License:        BSD-3-Clause
```

Comment:
Please change this to just say `BSD`, in order to align with the [Fedora license short names](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing).